src/Utils.ts

```diff
@@ -156,6 +156,7 @@ export function getProvider(): providers.Provider {
   );
 }
 
+//TODO improve the validating and type handling
 export const deployTransactionRequestShape = {
```

Comment:
The `RelayClient` send the `envelopingTx` over the internet and the `RelayServer` receive it, right now everything is being sent as string and validated using ow, but could use a different approach.

Reply:
I think we agreed on keeping this approach for now